final_task/rss_reader/functions/print_func.py

```diff
@@ -27,7 +28,7 @@ def generate_news_json(news_collection, logger):
     all_news_dict = {'news': []}
     logger.info("Configure json file...")
     for news in news_collection:
-        news_dict = asdict(news)
+        news_dict = news.asdict()
```

Comment:
better rename it to `as_dict` so it does not match with `asdict` from `dataclass` lib

Reply:
Fixed. Don't need it anymore